src/Home/Home.tsx

```diff
@@ -1,27 +1,35 @@
-import { useState } from "react";
+import React, { useState } from "react";
+import DisplayBgImage from '../images/DisplayBgImage';
 
 function Home() {
 
-    const [username, setUsername] = useState(""); 
-    const [submitStatus, setSubmitStatus] = useState(false);  
+    const [username, setUsername] = useState("");
+    const [submitStatus, setSubmitStatus] = useState(false);
 
     const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
         event.preventDefault();
         setSubmitStatus(true);
     }
 
+    const homeBgImg = DisplayBgImage();
```

Comment:
when we name things, we want to make it as clear as possible what that thing is/is doing, so it's best to leave out abbreviations and write out full words instead.
I would suggest renaming this to `homeBackgroundImage`